testData/solutions/win32.json5

```diff
@@ -2609,495 +2834,13473 @@
+                            "type": "string"
+                        }
+                    },
+                    "KeyLabels.Decimal": {
+                        "schema": {
+                            "title": "Manage Decimal Keylabel",
+                            "description": "Change Keylabel and Muting.",
+                            "type": "string"
+                        }
+                    },
+                    "KeyLabels.Delete": {
+                        "schema": {
+                            "title": "Manage Delete Keylabel",
+                            "description": "Change Keylabel and Muting.",
+                            "type": "string"
+                        }
+                    },
+                    "KeyLabels.Divide": {
+                        "schema": {
+                            "title": "Manage Divide Keylabel",
+                            "description": "Change Keylabel and Muting.",
+                            "type": "string"
+                        }
+                    },
+                    "KeyLabels.Down": {
+                        "schema": {
+                            "title": "Manage Down Keylabel",
+                            "description": "Change Keylabel and Muting.",
+                            "type": "string"
+                        }
+                    },
+                    "KeyLabels.End": {
+                        "schema": {
+                            "title": "Manage End Keylabel",
+                            "description": "Change Keylabel and Muting.",
+                            "type": "string"
+                        }
+                    },
+                    "KeyLabels.ErEOF": {
+                        "schema": {
+                            "title": "Manage ErEOF Keylabel",
+                            "description": "Change Keylabel and Muting.",
+                            "type": "string"
+                        }
+                    },
+                    "KeyLabels.Escape": {
+                        "schema": {
+                            "title": "Manage Escape Keylabel",
+                            "description": "Change Keylabel and Muting.",
+                            "type": "string"
+                        }
+                    },
+                    "KeyLabels.Execute": {
+                        "schema": {
+                            "title": "Manage Execute Keylabel",
+                            "description": "Change Keylabel and Muting.",
+                            "type": "string"
+                        }
+                    },
+                    "KeyLabels.ExSel": {
+                        "schema": {
+                            "title": "Manage ExSel Keylabel",
+                            "description": "Change Keylabel and Muting.",
+                            "type": "string"
+                        }
+                    },
+                    "KeyLabels.F1": {
```

Comment:
They lost me at "and muting".  If it was just a label field, fine, but how is the muting indicated?  Is it another combo field, pipe delimited?  Is there a special value that indicates that a key is muted?  Obviously if we plan to support this field we need at least basic details.

Reply:
These have been commented out due to lack of a default value.